db/migrate/20191118150004_create_activities.rb

```diff
@@ -0,0 +1,22 @@
+class CreateActivities < ActiveRecord::Migration[6.0]
+  def change
+    create_table :activities, id: :uuid do |t|
+      t.string :identifier
```

Comment:
We should _probably_ add a uniqueness constraint here as it's quite important that it's universally unique for IATI.